operations/service.go

```diff
@@ -165,6 +165,11 @@ func Service() cli.Command {
 				return errors.WithStack(err)
 			}
 
+			if err := model.CheckIndexes(ctx, env.GetDB(), model.GetRequiredIndexes()); err != nil {
```

Comment:
Can we have an option to skip this? Running cedar locally for testing would fail since your local DB probably doesn't have these indexes.

Reply:
Done. I moved the check for the DB URI.